xbmc/settings/dialogs/GUIDialogSettingsBase.cpp

```diff
@@ -521,13 +521,20 @@ std::set<std::string> CGUIDialogSettingsBase::CreateSettings()
     const CSettingControlTitle *title = dynamic_cast<CSettingControlTitle *>((*groupIt)->GetControl());
     bool hideSeparator = title ? title->IsSeparatorHidden() : false;
```

Comment:
Has checked, this looks a bit more easy, has added a temporary there: 
https://github.com/AlwinEsch/kodi/commit/19c4ebcb4d7cd517bc3776ed89744e432539ba24